### PR TITLE
Added ics files and updated Readme.md

### PR DESCRIPTION
**Pull request by eteridvalishvili**

### CODE REVIEW

**Review by minesh-s-patel — DISMISSED** [review]

**Comment by minesh-s-patel** [comment]
@iansloyan @tomhealey-icma @chrisisla 

Can one of you take a look and approve this PR? It looks good to me - but should be reviewed by at least one on you.

Thanks

File: .github/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+---
+Id: code-of-conduct
+Title: Community Code of Conduct
+---
+
+## Overview
+
+The Fintech Open Source Foundation ("FINOS" or the "Foundation") community is comprised of professionals and volunteers from all over the world, working in many different capacities including writing code, leading working groups, managing events, and more.
+
+Diversity is one of our strengths, but it can sometimes give rise to miscommunication and conflict. To ensure that all of our contributors feel welcome and valued, our code of conduct establlishes clear expectations for participants in the Foundation's work. It is not intended to serve as an exhaustive list of prohibited behavior, but rather, a set of principles for maintaining a healthy and inclusive community.
```
Comment (chrisisla) [inline .github/CODE_OF_CONDUCT.md:10]: "establishes"

File: .github/PULL_REQUEST_TEMPLATE/default.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+name: "Enrollment in the specification process"
+about: Create this pull request to abide to the terms to participate to this project
```
Comment (chrisisla) [inline .github/PULL_REQUEST_TEMPLATE/default.md:3]: should be "participate in this project"?

File: .github/PULL_REQUEST_TEMPLATE/default.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+title: Enrollment in the specification process
+labels: enrollment
+---
+
+My name is <FirstName LastName> and I'd like to formally participate to the <projectName> project.
```
Comment (chrisisla) [inline .github/PULL_REQUEST_TEMPLATE/default.md:8]: same here as well, "participate in"?

**Review by minesh-s-patel — APPROVED** [review]

**Pull request merged**